src/main/steps/applicant1/check-your-conditional-order-answers/template.njk

```diff
@@ -6,6 +6,7 @@
 {% set nextIncompleteStep = getNextIncompleteStepUrl() %}
 {% set isConditionalOrderAnswersComplete =
         '/check-your-conditional-order-answers' in nextIncompleteStep or
+        nextIncompleteStep === '/applicant2/' or
```

Comment:
is there a page which is just '/applicant2/'?

Reply:
There isn't but I double checked and this is needed for app2 as the getNextIncompleteStepUrl returns '/applicant2' for them when they finish the sequence